assets/blocks/quiz/data.js

```diff
@@ -81,16 +81,34 @@ export function parseQuestionBlocks( blocks ) {
 		.filter( ( block ) => !! block.title );
 }
 
+/**
+ * Create a new question block.
+ *
+ * @param {Object} item Question item.
+ *
+ * @return {QuizQuestion} Block.
+ */
+export function createQuestionBlock( item ) {
+	const { description, ...question } = item;
+
+	const innerBlocks =
+		( description && rawHandler( { HTML: description } ) ) || [];
+
+	const attributes = createQuestionBlockAttributes( question );
+
+	return createBlock( questionBlock.name, attributes, innerBlocks );
+}
+
 /**
  * Find a question block based on question ID, or title if ID is missing.
  *
  * @param {Array}        blocks
  * @param {QuizQuestion} item
  */
-const findQuestionBlock = ( blocks, { id, title } ) => {
+export const findQuestionBlock = ( blocks, { id, title } ) => {
 	const compare = ( { attributes } ) =>
 		id === attributes.id ||
-		( ! attributes.id && attributes.title === title );
+		( ! attributes.id && attributes.title && attributes.title === title );
```

Comment:
@yscik This seemed safe, but let me know if this is dangerous.  Otherwise it would match question blocks that were empty when I was searching for just `id`.

Reply:
Yup that should work out.